operator/src/main/java/oracle/kubernetes/weblogic/domain/model/ServerPod.java

```diff
@@ -492,6 +493,17 @@ private void addAuxiliaryImage(ServerPod serverPod1) {
     }
   }
 
+  private V1Container createWithEnvCopy(V1Container c) {
```

Comment:
I'm nervous about this as it will be difficult to remember to keep this in sync if later versions of K8s add more fields to V1Container. Can you use new V1ContainerBuilder(V1Container)? We should probably update the copy of V1EnvVar too.

Reply:
Change to use V1ContainerBuilder(V1Container) and V1EnvVarBuild(V1EnvVar) in c1cda3c. Please let me know if I missed something. Thanks.